.github/workflows/manual-upgrade-testnet-l2.yml

```diff
@@ -230,9 +230,9 @@ jobs:
           curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/obscuro-test/dispatches --data '{ "event_type": "dev_testnet_upgraded", "client_payload": { "ref": "${{ github.ref_name }}" }'
 
       - name: 'Send a repository dispatch to obscuro-test on upgrade of testnet'
```

Comment:
Would be good to make the comments consistent with the code also

Reply:
```suggestion
      - name: 'Send a repository dispatch to obscuro-test on upgrade of uat-testnet'
```